documentation/webCode/css/centered.css

```diff
@@ -0,0 +1,7 @@
+.centered {
+  height: 12em;
+  display: flex;
+  align-items: center;
+  justify-content: center }
+.centered p {
+  margin: 0 }
```

Comment:
Is this correct? Don't you want to only set the margin right and left?

Reply:
This is the one of the World Wide Web Consortium's examples on how to center things in CSS. I have no idea how CSS actually works but I needed something that worked with `span` and not `div` as `div` leads to all markdown being ignored by the markdown compiler.
I removed the height setting and that actually fixed the issue I had with the image on the PaperMC page. 
If you know any better way, let me know.

Reply:
What i mean is:
```css
margin-right: 0;
margin-left 0;
```

This would remove possible horizontal margins without changing any vertical margins and as far as I can see that should be what you intended here.